tensorflow/compiler/xla/service/gpu/llvm_gpu_backend/gpu_backend_lib.cc

```diff
@@ -558,6 +560,94 @@ static std::vector<string> GetROCDLPaths(int amdgpu_version,
   return result;
 }
 
+// Emits the given module to HSA Code Object. target_machine is an initialized
+// TargetMachine for the AMDGPU target.
+StatusOr<std::vector<uint8>> EmitModuleToHsaco(
+    Module* module, llvm::TargetMachine* target_machine) {
+  char tempdir_template[] = "/tmp/amdgpu_xla-XXXXXX";
+  char* tempdir_name = mkdtemp(tempdir_template);
+
+  VLOG(1) << "Compile-time artifacts located at: " << tempdir_name;
+
+  // prepare filenames for all stages of compilation:
+  // IR, ISA, binary ISA, and HSACO
```

Comment:
The code below sets up filenames for isabin (binary ISA) but not for ISA alone.

Reply:
will revise the comment.